topics/animation/index.md

```diff
@@ -4,4 +4,4 @@ short_description: Animation is the art of creating moving images.
 topic: animation
 wikipedia_url: https://en.wikipedia.org/wiki/Animation
 ---
-Animation is a method in which figures are manipulated to appear as moving images.
+Animation is a method in which figures are manipulated to appear as moving images. Illusory movies or moving images are created with the help of individual images, icons, etc.
```

Comment:
`Illusory` is a somewhat esoteric word IMO. I don't think that this sentence adds any more information or context to the existing description.